ml-agents/mlagents/trainers/stats.py

```diff
@@ -77,6 +79,9 @@ def write_stats(
 class ConsoleWriter(StatsWriter):
     def __init__(self):
         self.training_start_time = time.time()
+        # If self-play, we want to print ELO instead of reward
```

Comment:
It's in addition to - I didn't change how the ghosted agent adds the reward (it's still done by the AgentProcessor/Manager). 

Currently the trainer's "Extrinsic Reward" log (under Policy in TB) is what we want. We can do a hack in the ConsoleWriter to output Extrinsic Reward instead of Cumulative Reward (meaningless) as the reward if using self-play

Reply:
How would you feel about that? It could resolve this https://jira.unity3d.com/browse/MLA-760 in part.